bigtop-deploy/vm/vagrant-puppet-vm/README.md

```diff
@@ -56,13 +56,18 @@ num_instances: 5
 
 5) Test on local built packages is available by:
 
-first, build up local yum repo
```

Comment:
The sequence of the YUM build has changed indeed after some recent refactorings. Now it is
- ./gradlew componentname-pkg
- ./gradlew yum|apt